src/screens/login/index.tsx

```diff
@@ -50,10 +51,7 @@ const Login = ({navigation}: any) => {
         error={errors.password}
         type="password"
       />
-
-      <Button onPress={handleSubmit(onSubmit)} w={'50%'}>
-        Login
-      </Button>
+      <Button text="Login" handleSubmit={handleSubmit} onSubmit={onSubmit} />
```

Comment:
We should not manage handleSubmit and onSubmit by separate, you can create an unique prop called `handleSubmit` and then when you use the Button, you can pass the prop like this: `handleSubmit={handleSubmit(onSubmit)}`

Reply:
Done.